### PR TITLE
rtti: symbol table to lookup structures

### PR DESCRIPTION
**Pull request by mikea**
With this it is now possible to fully handle recursive types.

### CODE REVIEW

**Comment by mikea** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by mikea** [comment]
recheck

**Comment by kentonv** [comment]
recheck

**Comment by kentonv** [comment]
Wait its commit message says *I* signed but the commit content says you? And it still failed?

**Comment by kentonv** [comment]
recheck

**Comment by kentonv** [comment]
@larcher We're still having trouble here. It actually managed to commit something but somehow still fails...

**Review by kentonv — APPROVED** [review]

**Review by jasnell — APPROVED** [review]

**Comment by jasnell** [comment]
recheck

**Comment by jasnell** [comment]
Grr CLAssistant is being annoying again. The requirement has obviously been met so I'm going to go ahead and override the check and merge.

File: src/workerd/api/api-rtti-test.c++ [hunk 1]
```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache 2.0 license found in the LICENSE file or at:
 //     https://opensource.org/licenses/Apache-2.0
 
+#include "src/workerd/api/actor.h"
```
Comment (harrishancock) [inline src/workerd/api/api-rtti-test.c++:5]: This might need to be `<workerd/api/actor.h>`.

**Comment by mikea** [comment]
yeah :(

https://github.com/cloudflare/workerd/pull/99/files

**Pull request merged**